cmd/analyse_test.go

```diff
@@ -0,0 +1,58 @@
+package cmd
```

Comment:
I know the version test is in the same package, but if you look at the suite test you'll see the package name is <package>_test, this ensure that only black box testing can occur i.e. you can only invoke against public functions or structs.  Suggest renaming to cmd_test and deal with any issues that occur.